app/assets/templates/layout/footer.html.slim

```diff
@@ -4,7 +4,12 @@ footer.container-fluid(ng-controller="FooterCtrl")
       small
         | Powered by&nbsp;
         a(ng-href="https://github.com/DynamoMTL/sprangular") Sprangular
-
+      p
+        small
+          a ng-click="changeLng('en')" English
```

Comment:
What would be nice is if we have this softcoded, and then data bound with a `ng-repeat`. Also let's create an `ng-include` for it, like `layout/locales.html.slim`